rpc/src/main/scala/proto/annotations.scala

```diff
@@ -40,7 +40,7 @@ package object protocol {
   class option(val name: String, val value: String, val quote: Boolean) extends StaticAnnotation
 
   @message
-  case class Empty()
+  case class Empty(s: String = "")
```

Comment:
what happens if `@message object Empty { val s: String = "" }` ?

Reply:
It'll be the same case as [here](https://github.com/frees-io/freestyle-rpc/pull/75#pullrequestreview-74429578), right? But IMHO it deserves a try